client/my-sites/site-settings/composing/markdown.jsx

```diff
@@ -0,0 +1,39 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+
+class Markdown extends PureComponent {
+	static propTypes = {
+		disabled: PropTypes.bool,
+		siteId: PropTypes.number,
+		translate: PropTypes.func,
+	};
+
+	render() {
+		const { disabled, siteId, translate } = this.props;
+
+		return (
+			<FormFieldset className="composing__markdown has-divider">
+				<JetpackModuleToggle
+					disabled={ disabled }
+					label={ translate( 'Write posts or pages in plain-text Markdown syntax' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **11** times:
`translate( 'Write posts or pages in plain text Markdown syntax.' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).